features/logging/cluster_logging_operator.feature

```diff
@@ -152,3 +152,70 @@ Feature: cluster-logging-operator related test
       | card_name | #{cb.card} |
     Then the step should succeed
     """
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33868
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments - Invalid Values
+    Given I use the "openshift-logging" project
+    And I register clean-up steps:
+    """
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    """
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer_Invalid.yaml"
+    When I run the :create client command with:
+      | f | cl_fluentd-buffer_Invalid.yaml |
+    Then the step should fail
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33793
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer.yaml"
+    And I create clusterlogging instance with:
+      | remove_logging_pods | true                   |
+      | crd_yaml            | cl_fluentd-buffer.yaml |
+    Then the step should succeed
+    And I wait for the "fluentd" config_map to appear up to 300 seconds
+    When I run the :extract admin command with:
+      | resource  | configmap/fluentd |
+      | confirm   | true              |
+    Then the step should succeed
+    And evaluation of `File.read("fluent.conf")` is stored in the :fluent_conf clipboard
+    And the expression should be true> cb.fluent_conf != nil
```

Comment:
I'd like check two points.
1) The buffer size is in cb.fluent_conf . rather than cb.fluent_conf != nil
2) The fluend pods can be running.

Reply:
@anpingli Added the check for buffer size and checking fluentd pods are running before checking the fluent.conf.